modules/representation_management/lib/representation_management/v0/pdf_constructor/base.rb

```diff
@@ -11,9 +11,9 @@ def initialize(tempfile)
           @template_path = nil
         end
 
-        def construct(data)
+        def construct(data, flatten: true)
```

Comment:
Please add comments about this method (like you added for the other methods) along with a note that flatten is only set to false during tests, and explain why.